be/src/exec/pipeline/aggregate/spillable_aggregate_blocking_source_operator.cpp

```diff
@@ -49,6 +49,9 @@ bool SpillableAggregateBlockingSourceOperator::has_output() const {
     if (_aggregator->spiller()->has_output_data()) {
         return true;
     }
+    if (!_aggregator->spiller()->task_status().ok()) {
+        return true;
+    }
```

Comment:
we have already checked task_status in internal Spiller's internal interface, why check it here?

Reply:
when restore failed, has_output will be false. the query will be stuck